### PR TITLE
Add guidance on correcting a bid value

### LINKED ISSUE #155: Bids: Corrected price
Indonesia's [LPSE](https://lpse.lkpp.go.id/eproc4/) system collects structured data on 'bid prices' (the value originally submitted by the bidder) and 'corrected prices' (the confirmed bid value after any errors are corrected) ([example 1](https://lpse.lkpp.go.id/eproc4/evaluasi/7103119/pemenang), [example 2](https://lpse.lkpp.go.id/eproc4/evaluasi/7012119/pemenang)).

I am not sure of the exact scope of the errors that can be corrected in Indonesia's system, but the UNCITRAL model law Article 16 (2) states that:

> The procuring entity shall correct purely arithmetical errors that are discovered during the examination of submissions. 

The description of `Bid/value` doesn't specify which value should be provided, but it seems like the corrected price relates to the buyer's evaluation of the bid so, presumably, the value originally submitted by the bidder should go in `Bid/Value`.

For clarity, we should update the description of `Bid/Value`:

> The total value of the bid, as submitted by the bidder and before any corrections made during the assessment of bids.

That leaves the question of where to put the corrected price. Adding a sibling `Bid/correctedValue` field defined as follows seems like the simplest option:

> The total value of the bid, corrected for any purely arithmetical errors discovered during the assessment of bids.

@jpmckinney since you prefer not to mix the concepts of bid and bid assessment (CRM-6763) how would you suggest to model the corrected value?

cc @pindec

### PR DESCRIPTION
**Pull request by duncandewhurst**
Closes https://github.com/open-contracting/ocds-extensions/issues/155

### CODE REVIEW

**Review by mrshll1001 — APPROVED** [review]
This is good.

**Comment by jpmckinney** [comment]
I resolved the conflict by omitting the changelog entry about removing documents. A removal of irrelevant guidance is not very informative to a reader, so I don't think it needs to be logged.

File: README.md [hunk 1]
```diff
@@ -26,6 +26,19 @@ For example, publishers may wish to add statistics on minority or women-owned bu
 
 The codelist's Category column indicates whether the statistic applies to bids or bidders or whether it is specified or required by a particular regulatory context (e.g. EU).
 
+## Guidance
+
+### Correcting a bid value
+
+Procuring entities and, in some jurisdictions, bidders, can correct bid values after a bid is submitted; for example, to correct a purely arithmetic error or a misplaced decimal.
```
Comment (jpmckinney) [inline README.md]: https://ocds-standard-development-handbook.readthedocs.io/en/latest/meta/style_guide.html#word-choice

```suggestion
Buyers and procuring entities – and, in some jurisdictions, bidders – can correct a bid's value after the bid is submitted: for example, to correct an arithmetical error or a misplaced decimal mark.
```

File: README.md [hunk 2]
```diff
@@ -26,6 +26,19 @@ For example, publishers may wish to add statistics on minority or women-owned bu
 
 The codelist's Category column indicates whether the statistic applies to bids or bidders or whether it is specified or required by a particular regulatory context (e.g. EU).
 
+## Guidance
+
+### Correcting a bid value
```
Comment (jpmckinney) [inline README.md]:
```suggestion
### Correcting a bid's value
```

File: README.md [hunk 3]
```diff
@@ -26,6 +26,19 @@ For example, publishers may wish to add statistics on minority or women-owned bu
...
+
+Procuring entities and, in some jurisdictions, bidders, can correct bid values after a bid is submitted; for example, to correct a purely arithmetic error or a misplaced decimal.
+
+In OCDS:
+
+* The bid value as submitted goes in `bids.details.value`
+* If a correction is made, that bid value is overwritten
```
Comment (jpmckinney) [inline README.md]: The first bullet makes it sound like the field is *always* equal to the bid value as submitted. But that's only true at the time of submission.

Also, avoid metaphorical language like "goes in". (Can add this content to the standard handbook: https://ocp-software-handbook.readthedocs.io/en/latest/general/documentation.html#word-choice)

```suggestion
In OCDS, the bid's value is disclosed via the `bids.details.value` field. If a bid's value is corrected, the value of the `bids.details.value` field is overwritten. As such, the originally submitted value is only available via the contracting process' [change history](https://standard.open-contracting.org/latest/en/primer/releases_and_records/).
```

File: README.md [hunk 4]
```diff
@@ -26,6 +26,19 @@ For example, publishers may wish to add statistics on minority or women-owned bu
...
+
+To make both the original and corrected values available to data users, implementers can use the [releases and records model](https://standard.open-contracting.org/latest/en/primer/releases_and_records/) to publish a two releases: a release containing the original bid value and second release containing the corrected value.
```
Comment (jpmckinney) [inline README.md]:
```suggestion
As a publisher, to make both the original and corrected values available to users, publish at least two releases for the contracting process: one release containing the bid's originally submitted value and another containing its corrected value.
```

**Review by jpmckinney — APPROVED** [review]

**Pull request merged**